backend/README.md

```diff
@@ -0,0 +1,43 @@
+## RUNNING THE BACKEND SERVER
+
+This README provides essential information for setting up and running the server.ts file in this project.
+
+### Prerequisites
+
+Before running the server.ts file, make sure you have the following prerequisites installed:
+
+-   **Node.js:** Ensure that you have Node.js installed on your machine. You can download it from [Node](https://nodejs.org/).
+
+-   **TypeScript:** Install TypeScript globally using the following command:
+
+    ```bash
+    npm install -g typescript
+    ```
+
+-   **TypeScript dependencies:** You can install them using the following command:
+
+    ```bash
+    npm install --save-dev typescript @types/node
+    ```
+
+-   **Project dependencies:** You can install them using the following command:
+
+    ```bash
+    npm install
+    ```
+
+### Configuration
+
+-   Create a .env file in the root of your project and define the following variable:
+
+    ```env
+    PORT=your_desired_port_number
+    ```
+
+### Running the Server
+
+-   To run the server, use the following command:
+
+    ```bash
+    npx tsx server.ts
```

Comment:
u have created a script in package.json to run this. Is is not better to use the script though?

Reply:
You are right, changed to npm run server.

Reply:
True, my bad. I changed to npm run server.